projects/testing/core/marker-icon.harness.ts

```diff
@@ -0,0 +1,5 @@
+import {TuiComponentHarness} from '@taiga-ui/testing/utils';
+
+export class TuiMarkerIconHarness extends TuiComponentHarness {
```

Comment:
You should add it to `index.ts` too.

Reply:
I wasn't sure because as it is it's a really barebones harness for MarkerIcon and only used internally by the Action harness, but sure, I guess it still makes sense to export it too as it already provides basic harness

Reply:
done

Reply:
So far tests are not released as a separate library so we can introduce breaking changes, besides having a barebone harness  is better than none because handles necessary for further testing can be easily introduced.